qa/workunits/cephtool/test.sh

```diff
@@ -197,6 +197,12 @@ function ceph_watch_wait()
     fi
 }
 
+function test_mon_osd_modify_not_exsited_option()
+{
+  ceph daemon osd.0 config set aaaaa 1 | grep "no such an option"
+  ceph daemon mon.a config set aaaaa 1 | grep "no such an option"
```

Comment:
@tchaikov , I read the code in CephContext::do_command, and found it had no return value. If I add a return value for it, maybe big change... How about I just leave this unittest as it is now, and create a ticket in tracker?